e2e/test/helpers/RetryOperationHelper.cs

```diff
@@ -20,12 +20,53 @@ public class RetryOperationHelper
         /// This is for E2E tests of provisioning service clients.
         /// </remarks>
         /// <param name="asyncOperation">The async operation to be retried.</param>
-        /// <param name="retryPolicy">The retry policy to be applied.</param>
+        /// <param name="retryPolicy">The retry policy of hub device/module to be applied.</param>
         /// <param name="cancellationToken">The cancellation token to cancel the operation.</param>
         /// 
         public static async Task RunWithRetryAsync(
```

Comment:
nit: you might also want to rename these methods to call out client/service so we won't need to inspect the params to know the difference